tests/Plotly.NET.Tests/HtmlCodegen/GeoMapCharts.fs

```diff
@@ -0,0 +1,259 @@
+﻿module Tests.GeoMapCharts
+
+open Expecto
+open Plotly.NET
+open TestUtils
+open Plotly.NET.GenericChart
+open System
+
+let basemapChart =
+    Chart.PointGeo([]) // deliberately empty chart to show the base map only
+    |> Chart.withMarginSize(0, 0, 0, 0)
+
+let moreFeaturesBasemapChart =
+    let myGeo =
+        Geo.init(
+            Resolution=StyleParam.GeoResolution.R50,
+            ShowCoastLines=true, 
+            CoastLineColor="RebeccaPurple",
+            ShowLand=true, 
+            LandColor="LightGreen",
+            ShowOcean=true, 
+            OceanColor="LightBlue",
+            ShowLakes=true, 
+            LakeColor="Blue",
+            ShowRivers=true, 
+            RiverColor="Blue"
+        )
+    Chart.PointGeo([])
+    |> Chart.withMap myGeo
+    |> Chart.withMarginSize(0, 0, 0, 0)
+
+let cultureMapChart =
+    let countryGeo =
+        Geo.init(
+            Visible=false, 
+            Resolution=StyleParam.GeoResolution.R50,
+            ShowCountries=true, 
+            CountryColor="RebeccaPurple"
+        )
+    Chart.PointGeo([])
+    |> Chart.withMap countryGeo
+    |> Chart.withMarginSize(0, 0, 0, 0)
+
+[<Tests>]
+let ``Geo vs mapbox charts`` =
```

Comment:
``Geo charts`` ? i know the 'vs' path comes from the docs page but i do not think it makes sense here

Reply:
Noted. I still have no idea what `vs` means in this context, but okay, will remove

Reply:
versus (both docs articles demonstrate the differences between Geo and MapBox maps and link each other